WalletWasabi/Blockchain/Transactions/Summary/KnownInput.cs

```diff
@@ -0,0 +1,23 @@
+using NBitcoin;
+using WalletWasabi.Blockchain.TransactionOutputs;
+
+namespace WalletWasabi.Blockchain.Transactions.Summary;
+
+public class KnownInput : IInput
+{
+	private readonly SmartCoin _coin;
+	private readonly Network _network;
+
+	public KnownInput(SmartCoin coin, Network network)
+	{
+		_coin = coin;
+		_network = network;
+	}
+
+	public virtual Money Amount => _coin.Amount;
+
+	public BitcoinAddress Address => _coin.ScriptPubKey.GetDestinationAddress(_network)!;
+	public double Anonscore => _coin.HdPubKey.AnonymitySet;
+	public uint Index => _coin.Index;
```

Comment:
For some strange reason I am not allowed to push to this PR. Please cherry-pick my fix of the input index here: https://github.com/nopara73/WalletWasabi/commit/01cad50641ca8ed3d82f59f322a914d3779616e3

Reply:
Fixed! Added you as collaborator to my fork. That's the reason why you couldn't push 😅